core/server/worker/src/main/java/alluxio/worker/grpc/UfsFallbackBlockWriteHandler.java

```diff
@@ -52,8 +52,7 @@
 public final class UfsFallbackBlockWriteHandler
     extends AbstractWriteHandler<BlockWriteRequestContext> {
   private static final Logger LOG = LoggerFactory.getLogger(UfsFallbackBlockWriteHandler.class);
-  private static final long FILE_BUFFER_SIZE =
-      ServerConfiguration.getBytes(PropertyKey.WORKER_FILE_BUFFER_SIZE);
+  private static final long FILE_BUFFER_SIZE = Constants.MB;
```

Comment:
Should this be a constant since we use the same value in both write handlers?

Reply:
done, moved to `AbstractWriteHandler.java`